app.py

```diff
@@ -62,25 +61,49 @@ def flights():
     return "/flights?budget=xxx&seg=x&origin1=xxx&dest1=xxx&date1=xxxx-xx-xx"
 
 
-def get_flights(raw_input):
-    qpx.get(raw_input)
 
+def get_flights(raw_input):
+    file = qpx.get(raw_input)
     logging.warning(qpx.data)
-    print(qpx.data, file=open('latest_query.log', 'w+'))
+    # re-formated JSON returned data
+    return_data = dict()
+    # only return top 50 flight data 
+    for x in range(50):
+        option = dict()
+        option['price%d' % x] = qpx.data['trips']['tripOption'][x]['saleTotal']
```

Comment:
We might want to follow the python coding style on naming those variables, for example `option['price_%d % x]`